README.md

```diff
@@ -1,7 +1,8 @@
-<img align="left" style="padding: 10px" width="120" height="120" src="https://raw.githubusercontent.com/datarootsio/databooks/main/docs/images/logo.png">
+<img alt="logo" width="120" height="120" src="https://raw.githubusercontent.com/datarootsio/databooks/main/docs/images/logo.png">
 
 # databooks
 [![maintained by dataroots](https://dataroots.io/maintained-rnd.svg)](https://dataroots.io)
+[![PyPI](https://img.shields.io/pypi/v/databooks)](https://pypi.org/project/databooks/)
```

Comment:
Nice. Do you think we could also add [python versions](https://img.shields.io/pypi/pyversions/databooks)? Or do you think it'd get too cluttered?

Reply:
i'd add it :D